vector/src/main/java/im/vector/fragments/VectorRoomDetailsMembersFragment.java

```diff
@@ -726,9 +739,15 @@ public void onNetworkError(Exception e) {
 
                     @Override
                     public void onMatrixError(final MatrixError e) {
-                        kickNext();
-                        if (null != getActivity()) {
-                            Toast.makeText(getActivity(), e.getLocalizedMessage(), Toast.LENGTH_SHORT).show();
+                        if (getRiotActivity() != null) {
+                            if (MatrixError.M_CONSENT_NOT_GIVEN.equals(e.errcode)) {
+                                getRiotActivity().getConsentNotGivenHelper().displayDialog(e);
```

Comment:
Is it normal that you don't call `getRiotActivity().runOnUiThread()`here?

Reply:
Not sure, just doing this for code coherence. I think we can remove all the runoniothread() call, but didn't want to do it now